apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -126,7 +126,7 @@ const StripeLinkSectionCodec = t.strict({
   theme: t.union([t.literal('light'), t.literal('dark')]),
   subtitle: t.string,
   icon: t.union([StripeLinkMUIIconCodec, t.null]),
-  buttonText: t.union([t.string, t.null]),
+  buttonText: t.array(CTAButtonSimpleCodec),
```

Comment:
Why did this become an array of buttons?

Have requirements changed?

If so, the name "buttonText" should be replaced with "ctaButtons" for consistency.